typescript/infra/scripts/deploy.ts

```diff
@@ -51,6 +51,8 @@ import {
 } from './agent-utils.js';
 import { getEnvironmentConfig, getHyperlaneCore } from './core-utils.js';
 
+type Config = object;
```

Comment:
This is essentially as generic as the `unknown` type before. I do appreciate the attempt but the type would need to be defined entirely and accurately for us to accept the submission

Reply:
Oh, okay